src/test/java/com/endava/internship/collections/StudentTest.java

```diff
@@ -0,0 +1,124 @@
+package com.endava.internship.collections;
+
+import org.junit.jupiter.api.Assertions;
```

Comment:
In StudentSetTest class, you use static imports for **assertions**, but here you access those as static methods. In order to keep consistency, I would suggest using one way. Personally, I do prefer static imports over static access.